src/related-items-and-comparisons/OutfitList.jsx

```diff
@@ -0,0 +1,70 @@
+import React, {useEffect, useState} from 'react';
+import OutfitCard from './OutfitCard.jsx';
+
+const OutfitList = (props) => {
+  // hook for array of outfits, starts with nothing by default
+  // items must be unique, so clicking does nothing if product is already in the list
+  // should always have an "add to list" card
+  const [outfit, updateOutfit] = useState([]);
+  const [currList, setCurrList] = useState(outfit);
+  const [currIndex, setCurrIndex] = useState(0);
+
+  const next = () => {
+    setCurrIndex(currIndex + 1); // this is changing what currIndex will be the next time the component renders
+    setCurrList(outfit.slice(currIndex + 1));
+  };
+
+  const prev = () => {
+    setCurrIndex(currIndex - 1);
+    setCurrList(outfit.slice(currIndex - 1));
+  };
+
+  const addToOutfit = () => { // need to replace with outfit from the current page
+    if (!outfit.includes(props.products[0])) {
+      // updateOutfit([...outfit, props.products[0].product]);
+      updateOutfit([...outfit, props.products[0]]);
+    }
+  };
+
+  const removeFromOutfit = (product) => {
+    var index = -1;
+    for (var i = 0; i < outfit.length; i++) {
+      if (outfit[i].product.id === product.id) {
+        index = i;
```

Comment:
You can also use [Array.prototype.findIndex](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/findIndex)